apps/music/js/db.js

```diff
@@ -122,7 +122,13 @@ function initDB() {
     App.showCurrentView(function() {
       reparsingMetadata = false;
       // Hide the  spinner once we've displayed the initial screen
-      document.getElementById('spinner-overlay').classList.add('hidden');
+      // Setting the value stops the animation and works around bug 962594.
+      // The setTimeout is a workaround for Bug 1166500.
+      document.getElementById('spinner').value = 0;
```

Comment:
I switched from setAttribute() to the simpler form, but forgot to switch everywhere.

I've got a busted test anyway, so will fix both of these now.